cmd/tstr/ctl_test_list_cmd.go

```diff
@@ -29,5 +30,7 @@ var ctlTestListCmd = &cobra.Command{
 }
 
 func init() {
+	ctlTestListCmd.PersistentFlags().StringVar(&ctlTestNamespace, "namespace", "", "The namespace to use.")
```

Comment:
You should be able to omit this in favour of the persistent flag defined @ the test subcommand level (https://github.com/nanzhong/tstr/blob/main/cmd/tstr/ctl_test_cmd.go#L14).